vars/sfdxBuildPipeline.groovy

```diff
@@ -39,7 +40,36 @@ def call(Map parameters = [:]) {
     
     pipeline {
         node {
-            
+
+            if (notificationChannel) {
+                stage("slack notification start") {
+                    def user;
+                    def userId;
+                    def userEmail;
+
+                    /**
+                    * https://plugins.jenkins.io/build-user-vars-plugin/
+                    */
```

Comment:
Can you add a comment here to explain why this plugin was chosen instead of Jenkins pipelines? I feel you answered that but would be good to have this in the code.

Reply:
I've added that as part of the code